_po/ja/reference/index.po

```diff
@@ -629,6 +629,16 @@ msgstr "  * [`pgroonga_crash_safer.log_level`パラメーター][pgroonga-crash-
 msgid "  * [`pgroonga_crash_safer.log_path` parameter][pgroonga-crash-safer-log-path]"
 msgstr "  * [`pgroonga_crash_safer.log_path`パラメーター][pgroonga-crash-safer-log-path]"
 
+msgid ""
+"  * [`pgroonga_standby_maintainer.naptime` parameter][pgroonga-standby-maintai"
+"ner-naptime]"
+msgstr ""
+"  * [`pgroonga_standby_maintainer.naptime` パラメーター][pgroonga-standby-maintainer"
+"-naptime]"
+
+msgid "    * Since 2.4.2."
```

Comment:
I think it would be better to add verb for "since".
Considering Japanese below, probably Added or enabled or recommended .

Since ＋ number is normally used only 創業●年.　 
（もし技術文書ではよく使う表現であれば無視してください）

Reply:
APIリファレンスだとこういうときはsinceなんですよー

たとえば↓みたいに各種ドキュメントツールはsinceを使っています。

* https://www.doxygen.nl/manual/commands.html#cmdsince
* https://developer-old.gnome.org/gtk-doc-manual/stable/documenting_symbols.html.en
* https://rubydoc.info/gems/yard/file/docs/Tags.md#since

Reply:
なるほど。
そういものなのですね！失礼しました。
そして賢くなりました。ありがとうございます。